samples/fabric/fabric-cli/.gitignore

```diff
@@ -16,5 +16,6 @@ config.json
 package-lock.json
 src/wallet-network1
 src/wallet-network2
+src/data/remoteNetworkUsers
```

Comment:
I think you need to create an empty folder and push it, for the commands to work, else they will throw error that folder doesn't exist. You can put the contents of the folder in gitignore, like `clients/src/main/resources/config/credentials/remoteNetworkUsers/*`

Reply:
The code already takes care of it. It check if the directory exists, and if not, first it creates one with the required name. So nothing to do here.